### PR TITLE
pass-import: update to 3.3.

### PR DESCRIPTION
**Pull request by adbrown101**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/pass-import/template [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-version=3.2
-revision=2
+version=3.3
+revision=1
 build_style=python3-module
 # This test requires gnome-keyring which requires running dbus
 make_check_args="--deselect=tests/imports/test_parse.py"
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools python3-pypandoc python3-requests python3-yaml
+ python3-zxcvbn"
```
Comment (sgn) [inline srcpkgs/pass-import/template:9]: Please switch `distfiles` to `PYPI_SITE` https://pypi.org/project/pass-import/#files, `pypandoc` is not very welcome in hostmakedepends

Comment (sgn) [reply srcpkgs/pass-import/template:9]: https://github.com/roddhjav/pass-import/issues/192

Comment (adbrown101) [reply srcpkgs/pass-import/template:9]: Yes will do this evening

Comment (adbrown101) [reply srcpkgs/pass-import/template]: Changed to PYPI_SITE and removed the makedoc from setup.py. There are no tests in these sources, so do I drop the checkdepends altogether?

File: srcpkgs/pass-import/template [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+python3-SecretStorage python3-pykeepass python3-requests python3-yaml lastpass-cli python3-zxcvbn"
 checkdepends="python3-pytest $depends"
 short_desc="Pass extension for importing data from most password managers"
 maintainer="Alan Brown <adbrown@rocketmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/roddhjav/pass-import"
-distfiles="https://github.com/roddhjav/pass-import/releases/download/v${version}/pass-import-${version}.tar.gz"
-checksum=d943df14dcfec063a6c881c315ce65813db6e105b14a9b0156a73dcf7e529b76
+distfiles="${PYPI_SITE}/p/pass-import/pass-import-${version}.tar.gz"
+# distfiles="https://files.pythonhosted.org/packages/source/p/pass-import/pass-import-3.3.tar.gz"
```
Comment (paper42) [inline srcpkgs/pass-import/template:17]: why is this comment here?

File: srcpkgs/pass-import/template [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+version=3.3
+revision=1
 build_style=python3-module
 # This test requires gnome-keyring which requires running dbus
 make_check_args="--deselect=tests/imports/test_parse.py"
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools python3-requests python3-yaml python3-zxcvbn"
 depends="pass python3-defusedxml python3-magic python3-cryptography
-python3-SecretStorage python3-pykeepass python3-yaml"
-checkdepends="python3-pytest $depends"
```
Comment (paper42) [inline srcpkgs/pass-import/template:11]: don't remove pytest

Comment (paper42) [reply srcpkgs/pass-import/template:11]: oh, the pypi tarball doesn't have tests, in that case, could you just add
```
# pypi tarball doesn't contain tests and the github release doesn't contain pre-built manpages (is this true?)
# <link to sgn's issue>
make_check=no
```

Comment (adbrown101) [reply srcpkgs/pass-import/template:11]: Yes the first iteration did build from github and it requires pandoc to build markdown man files

```
ls -l share/man/man1/
-rw-r--r-- 1 alan alan 20046 Sep 10 23:19 pass-import.md
-rw-r--r-- 1 alan alan  8795 Sep 10 23:19 pimport.md

**Pull request merged**